test/interface-tests.spec.js

```diff
@@ -517,20 +605,30 @@ describe('kubo-rpc-client tests against go-ipfs', () => {
         : [])
   })
 
+  /**
+   * FIXME: https://github.com/ipfs/js-kubo-rpc-client/issues/56
+   * I recommend we remove these tests from the kubo-rpc-client as we have a pinning-service client that should be used instead
+   *
+   * @see https://github.com/ipfs-shipyard/js-pinning-service-http-client
+   */
```

Comment:
thoughts?

Reply:
I'd keep it. Different use case: this library is command-agnostic RPC wrapper for Kubo. User expectation is `kubo-rpc-client` supports everything that is exposed via `/api/v0` (RPC) or CLI.